app/controllers/ManageGroupController.scala

```diff
@@ -236,44 +236,28 @@ class ManageGroupController @Inject()(
     )
   }
 
-  def showViewGroupTeamMembers(groupId: String): Action[AnyContent] = Action.async { implicit request =>
-    // TODO get the groups team members
-    withGroupForAuthorisedOptedAgent(groupId, (group: AccessGroup) => {
-    //      val teamMembers = group.teamMembers.map { maybeUsers: Set[AgentUser] =>
-    //        maybeUsers.toSeq
-    //          .map(x => UserDetails(None, None, Some(x.id), Some(x.name))
-    //          .map(x => TeamMember.fromUserDetails(x)))
-    //      }.getOrElse(Seq.empty[TeamMember])
-
-      Ok(
-        view_team_members(
-          Seq.empty,
-          group.groupName,
-          routes.ManageGroupController.showManageGroupTeamMembers(groupId).url
-      )).toFuture
-    })
-
-    // Gets all the team members :3
-    //    isAuthorisedAgent { arn =>
-    //      isOptedIn(arn) { _ =>
-    //        agentPermissionsConnector.getGroup(groupId).flatMap(
-    //          maybeGroup => maybeGroup.fold(groupNotFound)(group => {
-    //            withSessionItem[Seq[TeamMember]](GROUP_TEAM_MEMBERS_SELECTED) { maybeSelectedTeamMembers =>
-    //              groupService
-    //                .getTeamMembers(arn)(maybeSelectedTeamMembers)
-    //                .flatMap { teamMembers =>
-    //                  Ok(
-    //                    view_team_members(
-    //                      teamMembers.getOrElse(Seq.empty),
-    //                      group.groupName,
-    //                      routes.ManageGroupController.showManageGroupTeamMembers(groupId).url
-    //                    )).toFuture
-    //                }
-    //            }
-    //          })
-    //        )
-    //      }
-    //    }
+  def showExistingGroupTeamMembers(groupId: String): Action[AnyContent] = Action.async { implicit request =>
+    isAuthorisedAgent { arn =>
```

Comment:
You can replace lines 240 to 243 with
 withGroupForAuthorisedOptedAgent(groupId, (group: AccessGroup) => {

Reply:
if I do that the arn isn't accessible on line 249
` groupService.getTeamMembersFromGroup(arn)(Some(teamMembers)).flatMap(groupMembers =>`

Reply:
I think arn is in the AccessGroup itself

Reply:
group.arn

Reply:
ahh okay, thanks!